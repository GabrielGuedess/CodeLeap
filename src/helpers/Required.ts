export type Required<T> = {
  [P in keyof T]-?: T[P];
};
