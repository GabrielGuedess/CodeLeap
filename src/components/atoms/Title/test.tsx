import { renderWithTheme } from 'utils/tests/helpers';

import { Title } from '.';

describe('<Title />', () => {
  it('should render the heading', () => {
    // Arrange
    const { container } = renderWithTheme(<Title>Title</Title>);

    // Assert
    expect(container.firstChild).toMatchSnapshot();
  });
});
