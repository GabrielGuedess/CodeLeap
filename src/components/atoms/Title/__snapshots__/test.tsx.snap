// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render the heading 1`] = `
.c0 {
  font-size: 2.2rem;
  font-weight: 700;
  color: #000000;
}

<h2
  class="c0"
  color="primary"
>
  Title
</h2>
`;
